src/components/inputs/rich-text-input/button.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import AccessibleButton from '../../buttons/accessible-button';
+import vars from '../../../../materials/custom-properties';
+
+const Button = props => {
+  return (
+    <AccessibleButton
+      isDisabled={props.isDisabled}
+      label={props.label}
+      css={css`
+        background: ${props.isActive ? vars.colorAccent30 : vars.colorSurface};
```

Comment:
Nitpick: for new components/styles, can we already support themes from the beginning?